### DISCUSSION TITLE
Add _transaction_commit_time column to the change store metadata columns.

### DISCUSSION THREAD

**Discussion by baiyangtx** [apache/amoro#977] [Ideas]
For this issue #971 ,  the query on the last snapshot of the change store could get the historical records of a specific key, but it is still hard to get the history of the records after a specific timestamp. If we add the `_commit_time` column which is the commit time of the transaction to change store metadata columns the query could filter by `_commit_time` and get the history of the records after some timestamp.

**Comment by baiyangtx** [comment]
@wangtaohz @zhoujinsong @Aireed CC

**Reply by zhoujinsong** [reply]
Adding a  `_commit_time` field when querying ChangeStore can indeed make it more convenient for users to use and filter data.
How would you like to implement it?
